bindings/python/triplestore/interface.py

```diff
@@ -48,7 +48,15 @@ def serialize(self, destination=None, format='xml', **kwargs)
         """
 
     def query(self, query_object, **kwargs)
```

Comment:
In the interface, the return-type should ideally be defined as well

Reply:
It is documented on line 51-59

Reply:
I was not talking about documentation, but rather type hints